feat: добавил openapi swagger

Fiarr4ikDev opened the pull request.

Fiarr4ikDev committed:
feat: добавил openapi swagger

Fiarr4ikDev committed:
feat: мелкие правки

Fiarr4ikDev committed:
feat: добавил выборку всех записей для всех сущностей

Fiarr4ikDev committed:
fix: исправленно предупреждение для метода setActive всех сущностей

gobaz095 reviewed (changes requested):
А теперь нужно описание для всего писать :)

Velier12 commented:
Я надеюсь это описание для сваггера а не джавадоки?

gobaz095 commented:
> Я надеюсь это описание для сваггера а не джавадоки?
Конечно, сами ДТО тоже нужно описывать
https://stackoverflow.com/questions/69210490/how-to-annotate-dto-so-that-it-shows-up-in-swaggerui-schema
Сами аннотации
https://github.com/swagger-api/swagger-core/wiki/Swagger-2.X---Annotations

gobaz095 commented:
И я все также жду тесты с использование testcontainers. Хотя бы на контроллеры и репозитории.

Fiarr4ikDev committed:
fix: исправил предупреждение для метода setActive всех сущностей.
feat: добавил названия для схем DTO и названия для контроллеров.
feat: написал javadoc

Fiarr4ikDev committed:
feat: сделал тесты для репозитория и контроллеров
